assets/Discord-ServiceContract.md

```diff
@@ -0,0 +1,43 @@
+# Discord
+
+## Service Manager: _Chris Daniel_
+### Lifecycle: _Live_
+### Evolution: _Product_
+
+## Service Description
+_Discord is the easiest way to talk over voice, video, and text. Talk, chat, hang out, and stay close with other members of the Wardley Mapping Community._
+
+_Discord, operated by the foundation, will gather community members in one place and boost interaction and collaboration._
+
+_To create your account, follow [this link](https://discord.com/invite/upg2deYD3Z) and let the Discord app guide you._
+
+_If you need a new community channel, request one by opening issue in [this Github repository](https://github.com/Wardley-Mapping-Foundation/wmf-operating-model)._
+
+_When any assistance is needed, notify @admin or @moderator in a chat._
+
+
+## User Needs
+As a _newcomer_  
+I need _to ask a question about Wardley Mapping_  
+so that _I can move forward with my issue_
+
+As an _active community member_  
+I need _a place where I can meet mapping experts_  
+so that _I can discuss my vision for the mapping community_
+
+As a _potential workgroup manager_  
+I need _to have one discoverable place where I can meet with other working group members_  
+so that _the working group can easily assemble and communicate with each other_
+
+Metrics:
+ - _Discord metrics will be unlocked once we reach 500 members._
+
+
+## Service Measures
+ - _Discord metrics will be unlocked once we reach 500 members._
+
+## Service Metrics
+ - _Discord metrics will be unlocked once we reach 500 members._
+
+## Service Dependencies
+ - _-_
```

Comment:
"none"

Reply:
I think that we do have dependencies for Discord?

- @cdaniel are a single point of failure as server owner? if so I think we need to list you ... also availability of admins (@cdaniel and I think @juliusgb?)
- availability of moderators (@cdaniel, @juliusgb, @spurkis, @agvbergin)
- capability (i.e. moderation skill) of moderators (same people)   
- [Discord service availability](https://discordstatus.com/)
- [Code of conduct](https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/blob/main/CODE_OF_CONDUCT.md) and sub-dependency of email reporting route
  - `hello@wardleymap.org`
  - @agvbergin or @spurkis access / ability to reply to that email
  - Password manager
  - domain registration of wardleymap.org

Reply:
@agvbergin : remember, these are ___Service___ dependencies.  Yes, we need to think about **all** of the things you've mentioned, but some of them are internal to the service.  I.e:

- @cdaniel + moderators - will provide skills & capabilities as **part of** this service.  The Service Manager will have to think about how to prevent SPOFs, and ensure enough skilled people are available to run the service.
- Discord itself (the 3rd party service) - will be sourced & managed as **part of** this service.
- Email, DNS & Password Management - are valid service **dependencies**, as they will not be provided as part of this service.  We've not decided on these yet, maybe they're all different user needs that are part of a single "End User Compute" service?  Or they could be separate services?